modules/social_features/social_profile/social_profile.module

```diff
@@ -46,13 +46,49 @@ function social_profile_field_widget_form_alter(&$element, FormStateInterface $f
     case 'field_profile_phone_number':
       // @todo Remove this when rule for .form-tel elements will be added.
       $element['value']['#attributes']['class'][] = 'form-text';
+      $element['value']['#attributes']['autocomplete'][] = 'tel';
       break;
 
     case 'field_profile_address':
       // @todo Remove this when script for custom selects will be added.
       $element['country_code']['#attributes']['class'][] = 'browser-default';
       break;
+
+    case 'field_profile_nick_name':
+      $element['value']['#attributes']['autocomplete'][] = 'nickname';
+      break;
+
+    case 'field_profile_first_name':
+      $element['value']['#attributes']['autocomplete'][] = 'given-name';
+      break;
+
+    case 'field_profile_last_name':
+      $element['value']['#attributes']['autocomplete'][] = 'family-name';
+      break;
+
+    case 'field_profile_organization':
+      $element['value']['#attributes']['autocomplete'][] = 'organization';
+      break;
+
+    case 'field_profile_function':
+      $element['value']['#attributes']['autocomplete'][] = 'organization-title';
+      break;
+
+    case 'field_profile_gender':
+      $element['#attributes']['autocomplete'][] = 'sex';
+      break;
+  }
+
+  // Change autocomplete value for the profile date field.
+  if ($field_definition->getType() === 'datetime') {
+    if ($context['items']->getEntity()->bundle() === 'profile') {
+      $element['value']['#attributes']['autocomplete'] = 'bday';
+    }
+    else {
+      $element['value']['#attributes']['autocomplete'] = 'off';
+    }
```

Comment:
This probably needs to be more specific. A distribution user may add date fields to a profile that are not a birthday (e.g. their work anniversary).

Reply:
Yes, I agreed with you, but above the field, we have a `Birthday` label. I think it will be a strange case if the user fills in something else.

Reply:
Hi @agami4 

I think @Kingdutch meant that users can add other date type fields to a profile and your changes will be applied to all of those fields.